src/cata_utility.h

```diff
@@ -227,6 +227,16 @@ double convert_volume( int volume, int *out_scale );
  */
 double temp_to_celsius( double fahrenheit );
 
+/**
+ * Convert a temperature from degrees Fahrenheit to degrees Kelvin.
+ *
+ * @param fahrenheit Temperature in degrees F.
```

Comment:
Do you really this and the `@returns` line? The first comment line looks sufficient, it explains what the functions does, what it requires and what it returns.

Reply:
Thats a copy of the temp_to_celsius comment, should i remove it there as well?

Reply:
Yes. Or is it necessary?